### ISSUE TITLE
Empty Bolt Extension config causes array_merge error in Extensions page

### ISSUE THREAD

**Issue by michaelborn** [bolt/core#3158]
A brief description of the issue goes here.

Details
-------

| Question                                              | Answer
|------------------------------------------------------ | ------
| Relevant Bolt Version                                 | 5.0
| Install type                                          |  Composer install
| <abbr title="Backwards Compatibility">BC</abbr> Break | no
| PHP version                                           | 8.0.12 |
| Web server                                            | Nginx

### Bug summary

The Bolt Extensions page bombs when an extension has an empty config file.

### Steps to reproduce

1. Install `bolt-dragsort` extension via `composer require jeschek/dragsort`
2. Navigate to `/bolt/extension`.
3. Observe the error.

### Expected result

A working, navigable extension page. IMO, extensions should work with zero config.

### Actual result

![Screenshot from 2022-03-31 17-14-13](https://user-images.githubusercontent.com/8106227/161151430-e995beed-4940-4712-9951-e9a8b0c43d23.png)

### Workaround

Adding `foo: true` to `config/extensions/jeschek-dragsort.yaml` fixes the issue so that I can at least browse the Extensions page.

**Comment by nestordedios** [comment]
Hi @michaelborn ,

Thank you for reporting about this and opening a PR in the Extension's repo.

A new PR #3242 has been opened to avoid this issue in the future. 

Once this is merged and released you can update Bolt by running `composer update`

**Comment by michaelborn** [comment]
Thanks @nestordedios ! Legendary work on the PR as well! :smile:
